src/energyplus/ForwardTranslator/ForwardTranslateCoilHeatingWaterToAirHeatPumpEquationFit.cpp

```diff
@@ -124,6 +124,14 @@ namespace energyplus {
       }
     }
 
+    // Part Load Fraction Correlation Curve Name
```

Comment:
Should the PR description be updated for handling new part load fraction correlation curve fields? (similar to addition of crankcase)

Reply:
I clarified it in the OP.

```diff
Coil Latent Stuff:
-   * Move some values from unitary systems to child object
+   * Move some values from unitary systems to child object: maximum Cycling Rate, Heat Pump Time Constant, Fan Delay Time => get moved as fields for cooling coil or as the "Part Load Fraction Correlation Curve" in case of heating
```